apps/earn-protocol/app/page.tsx

```
@@ -7,14 +7,34 @@ import { VaultListViewComponent } from '@/components/layout/VaultsListView/Vault
+      arksList: vault.arks,
+    }),
+  )
+
+  const interestRatesResults = await Promise.all(interestRatesPromises)
+
+  const vaultsDecorated = decorateCustomVaultFields({
+    vaults,
+    systemConfig,
+    decorators: {
+      arkInterestRatesMap: interestRatesResults.reduce((acc, curr) => {
+        return { ...acc, ...curr }
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)